lib/normalize_grammar.re

```diff
@@ -33,9 +29,7 @@ let rec normalize_to_simple = (body: A.rule_body): (B.simple, list(A.rule)) => {
 and normalize_to_atom = (body: A.rule_body): (B.atom, list(A.rule)) => {
   switch (body) {
   | A.TOKEN => (B.TOKEN, [])
-  | A.SYMBOL(name) => (B.SYMBOL(name), [])
-  | A.STRING(name) => (B.SYMBOL(name), [])
-  | A.PATTERN(name) => (B.SYMBOL(name), [])
+  | A.SYMBOL(name) | A.STRING(name) | A.PATTERN(name) => (B.SYMBOL(name), [])
```

Comment:
no, a symbol and a string are very different.
STRING AND PATTERN should convert to TOKEN I think.

Reply:
ok